roles/tas_single_node/templates/manifests/rekor/backfill_redis.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+LAST_INDEX_FILE="/tmp/rekor_last_index"
+
+endIndex=$(curl -sS http://localhost:3001/api/v1/log | jq -r '.treeSize - 1')
+if [ "$endIndex" -lt 0 ]; then
+  echo "info: no rekor entries found"
+  exit 0
+fi
+
+if [ -f "$LAST_INDEX_FILE" ]; then
+  startIndex=$(cat "$LAST_INDEX_FILE")
+else
+  startIndex=0
+fi
+
+if [ "$startIndex" -gt "$endIndex" ]; then
+  startIndex=0
+fi
+
+if [ "$startIndex" -le "$endIndex" ]; then
+  podman run --rm --name backfill_redis \
```

Comment:
Can we use `--force` instead of `--rm`? That should ensure that the previous container always stays on the node (until the cron job runs next time, which will replace it), so if there's a failure, the user has time to capture logs.